lib/phoenix/router/scope.ex

```diff
@@ -40,6 +40,7 @@ defmodule Phoenix.Router.Scope do
       opts
       |> Keyword.get(:metadata, %{})
       |> Map.put(:log, Keyword.get(opts, :log, top.log))
+      |> Map.put_new(:log_module, plug)
```

Comment:
I don't think we should put this by default, so we generate less metadata. Then in the formatter we do this:

    metadata[:log_module] || plug

WDYT?

Reply:
I like the idea! 
will make the change